lib/beaker/hypervisor/aws_sdk.rb

```diff
@@ -811,7 +811,18 @@ def create_new_key_pair(region, pair_name)
       ssh_string = public_key()
       region.key_pairs.import(pair_name, ssh_string)
       kp = region.key_pairs[pair_name]
-      if kp.exists?
+
+      exists = false
+      for tries in 1..10
```

Comment:
The exponential backoff here seems a little high; @mullr have you seen how long it takes when this needs to wait? 2**10 is an extra 25 minutes from 2**8, which is about 8 minutes of waiting and retrying.